### DISCUSSION TITLE
Problem with GitHub Copilot

### DISCUSSION THREAD

**Discussion by JayNguyen168** [community/community#113739] [GitHub Education]
### Hi everyone,

I was approved to use GitHub student pack. But why i still see this
024-03-23 10:12:04.402 [info] [auth] Invalid copilot token: missing token: 403 
2024-03-23 10:12:04.531 [error] [default] Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as JayNguyen168."

**Comment by wayne-wang-1119** [comment]
Prob a gitch on Github's side. 403 means you are not approved. More details to trouble shoot can be found here:
https://stackoverflow.com/questions/77374105/invalid-copilot-token-missing-token-403

**Comment by samus-aran** [comment]
Hi @JayNguyen168  👋

Ensure you have a successful GitHub Education [verification status](https://education.github.com/discount_requests/pack_application) and have received the confirmation email that your Academic benefits have been granted - this usually arrives 72 hours after you have been approved.

To enable Copilot, go to your account settings and under Code, planning, and automation section, select Copilot. You'll see "You're eligible to use copilot for free" and a button to claim the offer.

If you still unable to enable Copilot please then log out and log back in - prompting Copilot to re-authenticate. If all else fails raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "I am a verified student having trouble redeeming my Global Campus offers". 👍

Hope this helps! Let us know how you get along 👍
